### DISCUSSION TITLE
BattleBoxes Multiplayer Release 1.0.0

### DISCUSSION THREAD

**Discussion by spsquared** [spsquared/BattleBoxes_Multiplayer#54] [Releases]
BattleBoxes Multiplayer is now out of testing and you can enjoy the full BattleBoxes experience! The game has come a long way since its original creation on September 5th of 2020. It has had numerous clones and hopefully, you have been enjoying it. We have enjoyed making this game so far and will continue updating the game until it dies. It's amazing what we've done so far.

Features:
 - Account system
 - Platformer-style physics (with wall jumps!)
 - Point-round system
 - Achievements
 - The ever-important settings!

[Go Subscribe to Radioactive64!](https://www.youtube.com/channel/UChK4S12w_jjJs_c-jQh8WMQ/)

<hr /><em>This discussion was created from the release <a href='https://github.com/definitely-nobody-is-here/BattleBoxes_Multiplayer/releases/tag/v1.0.0'>BattleBoxes Multiplayer Release 1.0.0</a>.</em>

**Comment by SampleProvider** [comment]
more like

one commit on September 5th
